webview/src/views/main.svelte

```diff
@@ -1,12 +1,17 @@
 <script lang="ts">
+  import FileInput from "../components/file-input.svelte"
   /**
    * Event listener for starting a session from workflow ID
    */
-  function startFromWorkflowId(e: Event) {
-    if (!(e.target instanceof HTMLFormElement)) {
+  function startFromWorkflowId() {
+    const form = document.getElementById("debug-by-id-form")
+    if (!(form instanceof HTMLFormElement)) {
       throw new TypeError("Expected form element")
     }
-    const data = Object.fromEntries(new FormData(e.target))
+    const data = Object.fromEntries(new FormData(form))
+
+    // TODO: handle required fields on submit
+    if (!data?.workflowId) return
```

Comment:
Should not happen if input is `required`.

Reply:
This has been removed in [1c79ea5](https://github.com/temporalio/vscode-debugger-plugin/pull/18/commits/1c79ea5ac64702549a57c0338d9feab48a521e50) since the new `SubmitButton` allows for default form behavior.